docs/writing_tests.rst

```diff
@@ -160,7 +160,11 @@ Multiple workflows can use the same custom test like this:
 points to the folder where the named workflow was executed. This allows writing
 of advanced python tests for each file produced by the workflow.
 
+Custom tests must follow the `conventions for Python test discovery
+<https://docs.pytest.org/en/7.2.x/explanation/goodpractices.html#conventions-for-python-test-discovery>`_,
```

Comment:
This statically codes the link to the 7.2.x documentation. Please find the link for the dynamically updating lastest stable version of pytest.

Reply:
Done.